ntbs-service/Models/Notification.cs

```diff
@@ -55,6 +55,8 @@ public Notification()
         public virtual VisitorDetails VisitorDetails { get; set; }
         public virtual DenotificationDetails DenotificationDetails { get; set; }
         public virtual ComorbidityDetails ComorbidityDetails { get; set; }
+        [MaxLength(150)]
+        public string DeletionReason { get; set; }
```

Comment:
Does it not have a character set to validate against?
Also I would put it next other properties that are directly on this table, so `CreationDate` etc.